src/Acquisition/tests.py

```diff
@@ -1620,25 +1655,31 @@ def test_cant_persist_acquisition_wrappers_classic():
     >>> pickler = cPickle.Pickler(file, 1)
 
     >>> def persistent_id(obj):
+    ...     if not hasattr(obj, '_p_oid'): return None
```

Comment:
Er, of course not; no Persistent in sight. So I guess this is to deal with some difference in how pypy or Python 3 handles an AttributeError in persistent_id?